### PR TITLE
Release notes v1.2

### PR DESCRIPTION
**Pull request by shakalee14**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Spoke is an open source text-distribution tool for organizations to mobilize sup
 
 Spoke was created by Saikat Chakrabarti and Sheena Pakanati, and is now maintained by MoveOn.org.
 
-The latest version is [1.1.0](https://github.com/MoveOnOrg/Spoke/tree/v1.1) (see [release notes](https://github.com/MoveOnOrg/Spoke/blob/master/docs/RELEASE_NOTES.md#v11)) which we recommend for production use, while our `master` branch is where features still in development and testing will be available.
+The latest version is [1.2.0](https://github.com/MoveOnOrg/Spoke/tree/v1.1) (see [release notes](https://github.com/MoveOnOrg/Spoke/blob/master/docs/RELEASE_NOTES.md#v12)) which we recommend for production use, while our `master` branch is where features still in development and testing will be available.
```
Comment (schuyler1d) [inline README.md]: also update the link `https://github.com/MoveOnOrg/Spoke/tree/v1.1` even though it doesn't exist yet.  As soon as we merge, we will create it.

**Comment by mathemagica** [comment]
👍

**Review by mathemagica — APPROVED** [review]
👍

**Comment by shakalee14** [comment]
Merged with #500

**Pull request closed without merging**